src/main/java/anticope/rejects/modules/AutoEnchant.java

```diff
@@ -0,0 +1,130 @@
+package anticope.rejects.modules;
+
+import anticope.rejects.MeteorRejectsAddon;
+
+import meteordevelopment.meteorclient.events.game.OpenScreenEvent;
+import meteordevelopment.meteorclient.settings.*;
+import meteordevelopment.meteorclient.utils.network.MeteorExecutor;
+import meteordevelopment.meteorclient.utils.player.FindItemResult;
+import meteordevelopment.meteorclient.utils.player.InvUtils;
+import meteordevelopment.orbit.EventHandler;
+import net.minecraft.enchantment.EnchantmentHelper;
+import net.minecraft.item.Item;
+import net.minecraft.item.Items;
+import net.minecraft.screen.EnchantmentScreenHandler;
+import net.minecraft.screen.ScreenHandler;
+
+import java.util.List;
+import java.util.Objects;
+
+public class AutoEnchant extends meteordevelopment.meteorclient.systems.modules.Module {
+
+    public final SettingGroup sgGeneral = settings.getDefaultGroup();
+
+    private final Setting<Integer> delay = sgGeneral.add(new IntSetting.Builder()
+            .name("delay")
+            .description("The tick delay between enchanting items.")
+            .defaultValue(50)
+            .sliderMax(500)
+            .min(0)
+            .build()
+    );
+
+    private final Setting<Integer> level = sgGeneral.add(new IntSetting.Builder()
+            .name("level")
+            .description("Choose enchantment levels 1-3")
+            .defaultValue(3)
+            .max(3)
+            .min(1)
+            .build()
+    );
+
+    private final Setting<Boolean> drop = sgGeneral.add(new BoolSetting.Builder()
+            .name("drop")
+            .description("Automatically drops enchanted items (useful for when not enough inventory space)")
+            .defaultValue(false)
+            .build()
+    );
+
+    private final Setting<List<Item>> itemWhitelist = sgGeneral.add(new ItemListSetting.Builder()
+            .name("item-whitelist")
+            .description("Item that require enchantment.")
+            .defaultValue()
+            .filter(item -> item.equals(Items.BOOK) || item.isDamageable())
+            .build()
+    );
+
+    public AutoEnchant() {
+        super(MeteorRejectsAddon.CATEGORY, "auto-enchant ", "Automatically enchanting items.");
```

Comment:
trailing space in module name

Reply:
Thank you, we finally found the problem. No wonder we can't use commands `.toggle AutoEnchat` to toggle this module.